test/parallel/test-whatwg-url-invalidthis.js

```diff
@@ -10,7 +10,7 @@ const assert = require('assert');
   'toJSON',
 ].forEach((i) => {
   assert.throws(() => Reflect.apply(URL.prototype[i], [], {}), {
-    code: 'ERR_INVALID_THIS',
+    name: 'TypeError',
```

Comment:
If we're not testing the `code` value, we should test the `message` value.

Reply:
I've been thinking the same thing, but I'm concerned that this might lead to adjustments on v8 bumps.

Reply:
I'm sure it might, but this still needs to test that the correct error is thrown. That's one of the trade offs with not using code

Reply:
I updated the tests. Thanks for your time and your review